stock_vertical_lift_empty_tray_check/tests/test_pick.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo.addons.stock_vertical_lift.tests.common import VerticalLiftCase
+
+
+class TestPick(VerticalLiftCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.picking_out = cls.env.ref(
+            "stock_vertical_lift.stock_picking_out_demo_vertical_lift_1"
+        )
+        cls.env["ir.config_parameter"].set_param("vertical_lift_empty_tray_check", True)
```

Comment:
Use `sudo` in case the user is changed in base class (normally this tests should run w/ stock user)